src/reddit.py

```diff
@@ -12,6 +12,8 @@
 from nltk.corpus import wordnet as wn
 from utils import DB_DIR
 
+SCORE_THRESHOLD = -2
```

Comment:
Can you move this to `utils.py`? I would like to keep all the modifiable variables in the same place.